dispatcher/DAQController.py

```diff
@@ -43,6 +43,8 @@ def __init__(self, config, mongo_connector, log, hypervisor):
         self.time_between_commands = int(config['TimeBetweenCommands'])
         self.can_force_stop={k:True for k in detectors}
         self.one_detector_arming = False
+        self.start_cmd_delay = int(config['StartCmdDelay'])
```

Comment:
I personally would put this in a dictionary rather than two (or possibly more) growing list of delays. For two it doesn't matter but as soon as we have >3 parameters, I would use a dict.

Reply:
For sure, but we only have two values. If we decide on a reason to include more then a dict becomes a better idea.